src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/_help.py

```diff
@@ -40,4 +125,39 @@
 helps['appservice web config hostname list'] = """
     type: command
     short-summary: list all hostname bindings
+"""
+
+helps['appservice web browse'] = """
+    type: command
+    short-summary: Open the web app in a browser
```

Comment:
It seems all of the short summaries below should also refer to webapps like this one instead of webs.